### PR TITLE
fix: onWhatsApp query for multiple jids

### PR DESCRIPTION
**Pull request by arccoza**
The current query in `onWhatsApp` doesn't work when multiple `jid` args are provided (the request always times out).

### CODE REVIEW

**Comment by edgardmessias** [comment]
Thanks for your contribution.

The next step is to wait for review and approval to merge it to main repository

The community can help reacting with a thumb up (:thumbsup:) for approval and rocket (:rocket:) for who has tested it.

To test this PR you can run the following command below:
```
# NPM
npm install @whiskeysockets/baileys@arccoza/WhiskeyBaileys#fix-onwhatsapp
# YARN v2
yarn add @whiskeysockets/baileys@arccoza/WhiskeyBaileys#fix-onwhatsapp
```

**Review by edgardmessias — APPROVED** [review]

**Review by Fonnte — APPROVED** [review]

**Pull request merged**
